src/registry.ts

```diff
@@ -1,3 +1,5 @@
-import config from 'config';
+import config from "config";
 
-export const PREFIX = `${config.get('SchemaRegistry.Protocol')}://${config.get('SchemaRegistry.Hostname')}:${config.get('SchemaRegistry.Port')}/apis/registry`;
+export const PREFIX = `${config.get("SchemaRegistry.Protocol")}://${config.get("SchemaRegistry.Hostname")}:${config.get("SchemaRegistry.Port")}/apis/registry`;
```

Comment:
@jpramos123 are we sure that we want to keep this a constant? It uses config values, that seem like a moving part. Is config loaded before the constant is build? How is that ensured?

Reply:
I would prefer to keep this a constant. Also this is going to change when we are ready to use Apicurio in Clowder. Those values are going to be read from environment variables that are inside `clowdapp`.

Reply:
Again, how we'd ensure here that the config is loaded before the constant is created? This is a red flag, as it could likely have an impact in future (e.g. expected this to change based on config, but it would use outdated values).